### PR TITLE
user createdAt, updatedAt

### PR DESCRIPTION
**Pull request by patte**

### CODE REVIEW

File: packages/auth/graphql/resolvers/_queries/user.js [hunk 1]
```diff
@@ -13,11 +13,10 @@ module.exports = async (_, args, { pgdb, user: me }) => {
       : {username: slug}
   )
   if (
-    !user ||
-    (!user.hasPublicProfile && !Roles.userIsMe(user, me))
+    user &&
+    user.hasPublicProfile || Roles.userIsMeOrInRoles(user, me, ['admin', 'supporter'])
```
Comment (tpreusse) [inline packages/auth/graphql/resolvers/_queries/user.js]: wrap this line in parenthesis:
```js
> undefined && console.log(1) || true
true
```

File: packages/auth/graphql/resolvers/User.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 const Roles = require('../../lib/Roles')
+const userAccessRoles = ['admin', 'supporter', 'editor']
```
Comment (tpreusse) [inline packages/auth/graphql/resolvers/User.js]: IMO editor should be a general user access role—just email

File: packages/auth/graphql/resolvers/User.js [hunk 3]
```diff
@@ -20,5 +21,13 @@ module.exports = {
       return user.roles
     }
     return []
+  },
+  createdAt (user, args, { user: me }) {
+    Roles.ensureUserIsMeOrInRoles(user, me, userAccessRoles)
```
Comment (tpreusse) [inline packages/auth/graphql/resolvers/User.js]: For me it's unclear when we should throw based on field level access control. Who will consume `createdAt` and `updatedAt`? I can see use cases in the public profile for `createdAt`. Maybe just make it nullable and return null when not authorised?

File: packages/auth/graphql/resolvers/User.js (hunk 2 above)
Comment (patte) [reply packages/auth/graphql/resolvers/User.js]: agree and changed

File: packages/auth/graphql/resolvers/User.js (hunk 3 above)
Comment (patte) [reply packages/auth/graphql/resolvers/User.js]: disagree. IMHO the schema should represent reality. Every user has a createdAt and an updatedAt date. I think we should continue to treat access violations with throwing an exception. The client is responsible to only request the appropriate fields, in this case it's the admin tool.

File: packages/auth/graphql/resolvers/_queries/user.js (hunk 1 above)
Comment (patte) [reply packages/auth/graphql/resolvers/_queries/user.js]: absolutely

**Pull request merged**